src/Forces.jl

```diff
@@ -0,0 +1,66 @@
+using OrderedCollections
```

Comment:
Also not too convinced about this. Is this not a lot of boilerplate where a simple Dict or list could do as well. In hintsight I'm also not sure the `Energies` struct was a good decision.

Reply:
The custom show, display, etc. methods and the automatic calculation of total energies / forces are generally quite useful though. A more stripped-down `DecomposedQuantity` Dict-like struct that uses simple alphabetic ordering for pretty printing could be a cleaner way to do this?

Reply:
> The custom show, display, etc. methods and the automatic calculation of total energies / forces are generally quite useful though.
Yes good point.

> simple alphabetic ordering for pretty printing
Don't use alphabetic. The ordering as we have it was chosen to have energy terms align with the order of terms in the model. I would do the same for the forces (and use nothing or something for the terms without force contribs).

Yes a term-wise quantity structure would indeed be helpful I agree. One just has to make sure that one can easily AD through it (e.g. for stresses, sensitivities, ...). The more I think about it, the more I agree it might be worth introducing this, but that's a follow-up PR (and also probably a breaking interface change in the force computation functions).